app/src/main/java/com/framgia/moviedbtraining/movieDetails/PostersAdapter.kt

```diff
@@ -22,6 +24,10 @@ class PostersAdapter(private var movies: List<MovieDetails.Posters>,
               itemView.imageViewPoster)
         }
       }
+      itemView.imageViewPoster.setOnClickListener {
+        App.self().startActivity(Intent(App.self(), ImageFullActivity::class.java)
```

Comment:
should be like this 
https://viblo.asia/p/training-co-bao-nhieu-cach-de-start-mot-activity-bang-kotlin-android-LzD5dNq0ZjY
https://medium.com/@passsy/starting-activities-with-kotlin-my-journey-8b7307f1e460